app/finders/songs_finder.rb

```diff
@@ -0,0 +1,25 @@
+class SongsFinder
+  
+  def self.search(query)
+    return [] if query.blank?
+    
+    Song.find_by_sql <<-SQL.squish
+      SELECT
+        *,
+        MATCH (name) AGAINST ('>(#{query})' IN BOOLEAN MODE) +
```

Comment:
Should we sanitize the query before inserting it directly in raw SQL like this? Or are we okay because you put it in single quotes?